docker/build.sh

```diff
@@ -41,11 +45,11 @@ tag=${t:-latest}
 repository=${r:-"uniprimeroxacrdev.azurecr.io"}
 with_latest=${l:-false}
 
-BUILD_NGINX="docker build -f nginx/Dockerfile . -t primero/nginx:${tag} -t ${repository}/primero/nginx:${tag}"
-BUILD_BEANSTALKD="docker build -f beanstalkd/Dockerfile . -t primero/beanstalkd:${tag} -t ${repository}/primero/beanstalkd:${tag} --build-arg BEANSTALKD_PORT=${BEANSTALKD_PORT}"
-BUILD_SOLR="docker build -f solr/Dockerfile ../ -t primero/solr:${tag} -t ${repository}/primero/solr:${tag}"
-BUILD_APP="docker build -f application/Dockerfile ../ -t primero/application:${tag} -t ${repository}/primero/application:${tag} --build-arg APP_ROOT=${APP_ROOT} --build-arg RAILS_LOG_PATH=${RAILS_LOG_PATH}"
-BUILD_POSTGRES="docker build -f postgres/Dockerfile . -t primero/postgres:${tag} -t ${repository}/primero/postgres:${tag}"
+BUILD_NGINX="docker-compose build nginx"
```

Comment:
I don't want to build stuff using docker-compose: it's a fairly primitive tool. Compose should be used merely as an orchestration tool. Let's stick to vanilla docker builds. I am also suspicious of compose files taking on a dual task of orchestration and Makefile.

Reply:
I believe as well that tools should do one thing and do it well, but lots of current tools, like docker and kubectl do more things than its main purpose. And docker-compose is a well-maintained, stable and widely used tool. The documentation is fine with setting context, dockerfile locations and other build related parameters. It eases the maintainability, allows a developer running a "docker-compose up" get everything needed to get started.